src/utils/utils.cairo

```diff
@@ -678,4 +678,64 @@ namespace Helpers {
         let address = hash.low + r * 2 ** 128;
         return address;
     }
+
+    // @notice transform muliple bytes into words of 32 bits (big endian)
+    // @dev the input data must have length in multiples of 4
+    // @dev you may use the function `fill` to pad it with zeros
+    // @param data_len The length of the bytes
+    // @param data The pointer to the bytes array
+    // @param n_len used for recursion, set to 0
+    // @param n used for recursion, set to pointer
+    // @return n_len the resulting array length
+    // @return n the resulting array
+    func bytes_to_words_32bit_array{range_check_ptr}(
```

Comment:
Thanks for the review @ClementWalter ! Sorry I just got internet again, indeed it's `bytes_to_bytes4` and you're right in the context of the EVM a word is `bytes32` so I'll update this to avoid confusion 👍